migtests/tests/pg/omnibus/expected_files/expected_failed_2.25.sql

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
when compared to default expected_failed file, is there anything other than just different error messages?